fastchat/model/model_adapter.py

```diff
@@ -726,11 +729,11 @@ def load_model(self, model_path: str, from_pretrained_kwargs: dict):
         return model, tokenizer
 
     def get_default_conv_template(self, model_path: str) -> Conversation:
-        if "mpt-7b-chat" in model_path:
+        if "mpt-7b-chat" in model_path.lower():
```

Comment:
Please extract common expressions. Propagate this fix to other changes in this file. There are multiple similar places.
@surak Please always double-check your PR by visiting all your changes at https://github.com/lm-sys/FastChat/pull/2030/files